src/config.js

```
@@ -25,22 +26,23 @@ async function findService(dealId, headers) {
 }
 
 async function createConfig() {
-  const { token } = await inquirer.prompt([
-    { type: 'input', message: 'Productive.io token', name: 'token', type: 'password' },
-  ]);
+  const session = await createSession();
+
+  if (!session) {
```

I would move this if into the createSession, this way we "box" the features and fail in the module.

---

How should I handle exception there? I guess instead of returning null, I could throw an error of some kind?

---

Yeah, it's cli we just let it die. We'll worry about correct exit codes sometimes later.